coriolis/osmorphing/base.py

```diff
@@ -342,3 +343,28 @@ def _set_selinux_autorelabel(self):
                 "touch /.autorelabel")
         except Exception as err:
             LOG.warning("Failed to set autorelabel: %r" % err)
+
+    def _retain_user_credentials(self):
+        cloud_cfg_path = "/etc/cloud/cloud.cfg"
```

Comment:
Considering it's theoretically possible for these things we care about to be defined within separate files in '/etc/cloud/cloud.cfg.d/*.cfg', please make this method iterate through all files and apply the modifications to all of them.